spark-3.1-spanner-lib/src/test/java/com/google/cloud/spark/spanner/ReadIntegrationTestBase.java

```diff
@@ -58,39 +60,191 @@ public void testReadFromCompositeTable() {
     List<String> expectIds = Arrays.asList("id1", "id2");
     assertThat(gotIds).containsExactlyElementsIn(expectIds);
 
-    // 2. Validate C field string values.
+    // 2. Validate A field ARRAY<INT64> values.
+    List<Row> allAs = df.select("A").collectAsList();
+    assertEquals(allAs.size(), 2);
+    Row rowA1 = allAs.get(0);
+    assertEquals(rowA1.size(), 1);
+    Row rowA2 = allAs.get(1);
+    assertEquals(rowA2.size(), 1);
+    List<Long> gotAs = new ArrayList();
+    gotAs.addAll(rowA1.getList(0));
+    gotAs.addAll(rowA2.getList(0));
+    Collections.sort(gotAs);
+
+    long[] lA = {10, 100, 991, 567282, 20, 200, 2991, 888885};
+    List<Long> wantAFlattened = Arrays.stream(lA).boxed().collect(Collectors.toList());
```

Comment:
We don't need extra lA here. 

Arrays.stream({10, 100, 991, 567282, 20, 200, 2991, 888885}).boxed().collect(Collectors.toList())

Reply:
That code doesn't compile but using
```java
(new long[]{...})
```
though it is ugly which is why I did `long[] lA = {...}` but anyways I've gone with the latter.